pyproject.toml

```diff
@@ -1,10 +1,11 @@
 [project]
-name = "google-tasks-api"
```

Comment:
make sure you have mypy and ruff and all with all checks enabled like we asked in HW1
make sure all checks pass or carefully disable the ones that don't pass and don't make sense.